packages/fs-detectors/test/fixtures/40-rush-monorepo/apps/my-app/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "a",
```

Comment:
I think the name in each package.json should be unique

Reply:
Oh yes let me update it to match the path name (I had been doing that for most)